src/main/java/net/minecraftforge/common/util/Constants.java

```diff
@@ -49,4 +49,89 @@
         public static final int TAG_LONG_ARRAY  = 12;
         public static final int TAG_ANY_NUMERIC = 99;
     }
+
+    /**
+     * The world event IDS, used when calling World.playEvent.
+     * Can be found from RenderGlobal#playEvent
+     * Some of the events use the "data" parameter. If this is the case, an explanation of what "data" does is also provided
+     */
+    public static class WorldEvents {
+        public static final int DISPENSER_DISPENSE_SOUND = 1000;
+        public static final int DISPENSER_FAIL_SOUND = 1001;
+        /**
+         * Like DISPENSER_DISPENSE_SOUND, but for items that are thrown (arrows, eggs, snowballs)
```

Comment:
I'd change thrown to fired, as you don't really throw an arrow
```suggestion
         * Like DISPENSER_DISPENSE_SOUND, but for items that are fired (arrows, eggs, snowballs)
```